editor/src/absm/state_graph/mod.rs

```diff
@@ -210,6 +210,22 @@ impl StateGraphViewer {
                             }
                         }
                     }
+                    AbsmCanvasMessage::CommitTransitionToAllNodes {
+                        source_node,
+                        dest_nodes,
+                    } => {
+                        if message.direction() == MessageDirection::FromWidget {
+                            let source = fetch_state_node_model_handle(*source_node, ui);
+                            for dest_node in dest_nodes {
+                                let dest = fetch_state_node_model_handle(*dest_node, ui);
+                                sender.do_scene_command(AddTransitionCommand::new(
```

Comment:
Here you create a separate command per each transition, and if you try to undo this in the editor, you'll see that each of your transitions will be removed one-by-one. You should use command group instead. For example, when you dragging multiple nodes, the editor creates a group command for all nodes (https://github.com/FyroxEngine/Fyrox/blob/master/editor/src/absm/state_graph/mod.rs#L159). You could use this as an example.